etc/config/ada.amazon.properties

```diff
@@ -29,6 +29,43 @@ compiler.gnatsnapshot.objdumper=/opt/compiler-explorer/gcc-snapshot/bin/objdump
 compiler.gnatsnapshot.name=x86-64 gnat (trunk)
 compiler.gnatsnapshot.semver=(trunk)
 
+################################
+# GNAT for riscv64
+group.gnatriscv64.compilers=gnatriscv64112:gnatriscv64103
+group.gnatriscv64.groupName=GNAT riscv64
+group.gnatriscv64.instructionSet=riscv
+group.gnatriscv64.baseName=riscv64 gnat
+group.gnatriscv64.isSemVer=true
+group.gnatriscv64.adarts=foo
+
+compiler.gnatriscv64103.exe=/opt/compiler-explorer/riscv64/gnat-riscv64-elf-linux64-10.3.0-2/bin/riscv64-elf-gnat
+compiler.gnatriscv64103.semver=10.3.0
+compiler.gnatriscv64103.supportsBinary=false
+compiler.gnatriscv64103.adarts=/opt/compiler-explorer/riscv64/gnat-riscv64-elf-linux64-10.3.0-2/riscv64-elf/lib/gnat/zfp-rv64imc
```

Comment:
this is missing - I only see:
```
Jan/26 19:57 admin-node~/infra (main|✔) $ ls /opt/compiler-explorer/riscv64/gnat-riscv64-elf-linux64-10.3.0-2/riscv64-elf/lib/gnat
zfp-rv32i/  zfp-rv32iac/  zfp-rv32im/  zfp-rv32imac/  zfp-rv32imafc/  zfp-rv64imac/  zfp-rv64imafdc/                              
```

Reply:
Fixing it now

Reply:
Let me double check the 4 config before testing again (discord is KO for me)

Reply:
Should be good now. I've used nearly the same config locally and for the cross compiler only used a search/replace to change path prefix... and it works...